src/__tests__/exposure-provider.test.tsx

```diff
@@ -352,6 +356,33 @@ describe('useExposure', () => {
     });
   });
 
+  describe('pause()', () => {
+    it('pauses the exposure module', async () => {
+      const {result} = await renderExposureHook();
+      await act(async () => {
+        await result.current.pause();
+      });
+      expect(ExposureNotificationModule.pause).toHaveBeenCalledTimes(1);
+    });
+
+    it('validates the status', async () => {
+      const {result} = await renderExposureHook();
+      mocked(ExposureNotificationModule.status).mockClear();
+      await act(async () => {
+        await result.current.pause();
+      });
+      expect(ExposureNotificationModule.status).toHaveBeenCalledTimes(1);
+    });
+
+    it('does not throw', async () => {
```

Comment:
Not sure if not reading right - but this function does indeed throw

Reply:
Will fix